core/src/main/java/com/graphhopper/routing/profiles/EncodedValueLookup.java

```diff
@@ -11,4 +11,6 @@
     DecimalEncodedValue getDecimalEncodedValue(String key);
 
     StringEncodedValue getStringEncodedValue(String key);
+
+    boolean containsEncodedValue(String key);
```

Comment:
I needed a way to check if the EncodingManager supports/contains a encodedValue. Not sure if we should call this method supports or if there is a different way of doing this?

Reply:
Not sure about the naming. The implementation in the AbstractFlagEncoder could be done with the encodedValueLookup

Reply:
How about `supportsEncodedValue`?

Reply:
> The implementation in the AbstractFlagEncoder could be done with the encodedValueLookup

The issue with this is that we would return the values that are stored in the EncodingManager, but maybe not with/because of this encoder. I am not sure if this is important. For example:

```
EncodingManager em = new EncodingManager.Builder().addGlobalEncodedValues().addAllFlagEncoders("car,bike,foot").build();
CarFlagEncoder encoder = (CarFlagEncoder) em.getEncoder("car");

encoder.containsEncodedValue(TagParserFactory.BIKE_ACCESS) // => is true
```

For me this seems a bit weird. But maybe it seems weird to me because I am still thinking too much about the "old" system?

Reply:
This is not a big issue as we will deprecate the FlagEncoder or at least how they work at the moment :)

Reply:
> How about supportsEncodedValue?

Best would be just "supports(String key)" but this clashes with the existing stuff for the FlagEncoder ... but maybe there is no clash as the naming is quite different.

Reply:
Yes there is a conflict with `EncodingManager#supports(String encoder)`.

Reply:
I know but the naming of an encoder is different (always?) to the one of the EncodedValue? At the end "supports encoder" is a usage we should deprecate